src/validation.cpp

```diff
@@ -3521,16 +3521,18 @@ void PruneOneBlockFile(const int fileNumber)
             pindex->nUndoPos = 0;
             setDirtyBlockIndex.insert(pindex);
 
-            // Prune from mapBlocksUnlinked -- any block we prune would have
+            // Prune from m_blocks_unlinked -- any block we prune would have
             // to be downloaded again in order to consider its chain, at which
             // point it would be considered as a candidate for
-            // mapBlocksUnlinked or setBlockIndexCandidates.
-            std::pair<std::multimap<CBlockIndex*, CBlockIndex*>::iterator, std::multimap<CBlockIndex*, CBlockIndex*>::iterator> range = mapBlocksUnlinked.equal_range(pindex->pprev);
+            // m_blocks_unlinked or setBlockIndexCandidates.
+            std::pair<std::multimap<CBlockIndex*, CBlockIndex*>::iterator,
+                std::multimap<CBlockIndex*, CBlockIndex*>::iterator> range =
+                    g_blockman.m_blocks_unlinked.equal_range(pindex->pprev);
```

Comment:
Could use `const auto` instead of splitting this into three lines.

Reply:
Happy to do that, but thought that the explicit type declaration might be helpful since the value is so complicated. If you think `const auto` is better I'll change it.